tests/pgshovel/streams/states.py

```diff
@@ -137,63 +152,29 @@ def test_require_different_publisher(message):
 
     require_different_publisher(
         Committed(
-            uuid.uuid1().bytes,  # change the publisher
-            batch_identifier,
+            publisher=uuid.uuid1().bytes,  # change the publisher
+            batch_identifier=batch_identifier,
         ),
+        0,
         message,
     )
 
     with pytest.raises(InvalidPublisher):
         require_different_publisher(
             Committed(
-                message.header.publisher,
-                batch_identifier,
+                publisher=message.header.publisher,
+                batch_identifier=batch_identifier,
             ),
+            0,
             message,
         )
 
 
-def test_stateful_validator():
```

Comment:
The new implementation of `StatefulStreamValidator` assumes a pretty tight integration to the `interfaces.streams` code, and seems to be pretty well tested by `test_successful_transaction` down below, so I just removed this test.  If you think we still need it I can try adding it back - but it's unclear to me how it's behaviorally different than `test_successful_transaction`.